test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
```

Comment:
The function can go in the [test utils file](https://github.com/js-cookie/js-cookie/blob/4a4122eb06bc6c4fa8034f00dd9170bd07d4ce7c/test/utils.js).